config/dev.env

```diff
@@ -1,7 +1,7 @@
-CB_URL=localhost
-CB_USER=Administrator
-CB_PASS=password
-CB_BUCKET=user_profile
-IS_CAPELLA=false
+DB_CONN_STR=couchbases://<identifier>.cloud.couchbase.com
+DB_USERNAME=
+DB_PASSWORD=
+DB_BUCKET_NAME=travel-sample
+IS_CAPELLA=true
```

Comment:
Do you need the IS_CAPELLA anymore? 
Wan Profile will work with both on prem & capella. It is only setting a few timeouts.

Reply:
You are right, it is not required, removed the IS_CAPELLA flag.